### PR TITLE
[2.2] Prepare for minor release

### PR DESCRIPTION
**Pull request by rossriley**
Update changelog and version.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Changelog for Bolt 2.x
 ======================
 
+Bolt 2.2.24
```
Comment (bobdenotter) [inline changelog.md]: 2.2.25 :-)

Comment (rossriley) [inline changelog.md]: talk about failing at the basics...

**Comment by bobdenotter** [comment]
Disregard CI failures, acquire merge.

**Comment by GawainLynch** [comment]
> I'll package a release tomorrow!

I don't know we need anything more than a git tag, as this will only affect Composer installs.

**Pull request merged**